cocos/spine/lib/spine-define.ts

```diff
@@ -42,6 +43,50 @@ function overrideDefineArrayProp (prototype: any, getPropVector: any, name: stri
     });
 }
 
+function overrideDefineArrayArrayProp (prototype: any, getPropVector: any, name: string): void {
+    Object.defineProperty(prototype, name, {
+        get (): any[] {
+            const array: any[] = [];
+            const vectors = getPropVector.call(this);
+            const count = vectors.size();
+            for (let i = 0; i < count; i++) {
+                const vectorI = vectors.get(i);
+                const countJ = vectorI.size();
+                const arrayJ: any[] = [];
+                for (let j = 0; j < countJ; j++) {
+                    arrayJ.push(vectorI.get(j));
+                }
+                array.push(arrayJ);
+            }
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+            return array;
+        },
```

Comment:
Do we need to add setter for this?

Reply:
Currently there is only one place to use and only get.
<img width="503" alt="Screenshot 2023-08-23 at 10 39 34" src="https://github.com/cocos/cocos-engine/assets/2398658/fd9115d6-acb9-4895-93fb-d2660a37996a">
![Uploading Screenshot 2023-08-23 at 10.39.44.png…]()